mrchecker-framework-modules/mrchecker-cli-module/pom.xml

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <parent>
+        <artifactId>mrchecker-test-framework</artifactId>
+        <groupId>com.capgemini.mrchecker</groupId>
+        <version>3.3.0</version>
+    </parent>
+
+    <artifactId>mrchecker-cli-module</artifactId>
+    <version>1.0.1</version>
+    <packaging>jar</packaging>
+    <name>MrChecker - CLI - Module</name>
+    <description>MrChecker CLI Module supports:
+        * Running CLI applications on either Windows or Linux
+        * Reading from the stdout and stderr streams
+        * Writing to the stdin stream
+        * Single run or interactive mode
+        * The stdout stream is watched by a reader thread
+        * The stderr may be optionally watched by a reader thread
+        * Setting up env params
+    </description>
+    <url>https://github.com/devonfw/devonfw-testing</url>
+
+    <organization>
+        <name>Capgemini - Nearshore Test Center - Poland</name>
+        <url>http://nsc.capgemini.com/nearshoretestcenter</url>
+    </organization>
+
+    <developers>
+        <developer>
+            <name>Michal Babiarz</name>
+            <email>michal.babiarz@capgemini.com</email>
+            <organization>Capgemini - Nearshore Test Center - Poland</organization>
+            <organizationUrl>http://nsc.capgemini.com/nearshoretestcenter</organizationUrl>
```

Comment:
This page doesn't exist anymore

Reply:
This is present in all the modules. What do we want to do with that? Remove or replace with some other URL?

Reply:
https://capgeminisoftware.pl/